### ISSUE TITLE
Search in documentation site doesn't work

### ISSUE THREAD

**Issue by Tony20221** [AngleSharp/anglesharp.github.io#4]
Whatever word I search for at https://anglesharp.github.io/docs/01-articles?, I always get the same list of contents.

**Comment by FlorianRappl** [comment]
Yeah thanks for noticing - seems to be broken. We should update the docs.
